doc/user_guide/templates/prepare_original_checksum_template.md

```diff
@@ -0,0 +1,39 @@
+# Templates of `prepare_original_checksum.yml`
+
+Copy the content into a file `/.github/workflows/prepare_original_checksum.yml` in your project.
+
+## For Java Maven Project
+
+See [an example from this project](../../../.github/workflows/prepare_original_checksum.yml).
+
+## For Scala Sbt Project
+
+```
+name: Prepare Original Checksum
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+      - name: Assembly with SBT
+        run: sbt assembly
+      - name: Running tests
+        run: sbt test it:test
+      - name: Prepare checksum
+        run: find target/scala*/stripped -name *.jar -exec sha256sum "{}" + > original_checksum
+      - name: Upload checksum to the artifactory
+        uses: actions/upload-artifact@v2
+        with:
+          name: original_checksum
+          retention-days: 5
```

Comment:
Do we need this retention-days parameter? It seems like it removes the checksum.

Reply:
We discussed it via Slack, so I keep the parameter